lib/faexport.rb

```diff
@@ -400,6 +400,219 @@ def ensure_login!
       end
     end
 
+    # GET /new_submissions.json
+    get %r{/new_submissions\.(json|xml|rss)} do |type|
```

Comment:
Think this should be `/notifications/submissions` to match the other endpoints

Reply:
Does that also mean new_submissions.json should be rolled into notifications.json? Or just have that one separate for json/xml, like the rss ones all are?

Reply:
I'd be inclined to move `/notifications` to `/notifications/others` to match the FA endpoint and leave it the same

Reply:
Okay, so, lay it out like such:
- /notifications/submissions.(json|xml|rss)
- /notifications/others.(json|xml)
- /notifications/watches.rss
- /notifications/submission_comments.rss
- ....

Sounds good to me, I'll reorganise that tonight

Reply:
Moved to that format, updated docs, and updated index page (including adding new submissions endpoint to index page) in 5cbbb34a0aa1492e869530f1112aad211f27025a

Reply:
Yes I'm happy with it